src/actions/get-bounty-amount-updated-event.ts

```diff
@@ -0,0 +1,81 @@
+import db from "src/db";
+import {
+  BountiesProcessed,
+  EventsQuery,
+} from "src/interfaces/block-chain-service";
+import BlockChainService from "src/services/block-chain-service";
+import logger from "src/utils/logger-handler";
+
+export const name = "getBountyAmountUpdatedEvents";
+export const schedule = "1 * * * * *";
+export const description = "retrieving bounty created events";
+export const author = "clarkjoao";
+
+export async function getBountyAmountUpdate(
+  query?: EventsQuery
```

Comment:
What happens when `cma` calls this action _without any arguments_ ? (because that's how this function is supposed to be called)